### DISCUSSION TITLE
Does Gridify support IN List filtering

### DISCUSSION THREAD

**Discussion by asafblum** [alirezanet/Gridify#103] [General]
I really like to library, but cant find any operator like "IN"

Counties IN (US,CANADA) 

Can be implemented with OR but then very verbose on bigger Lists having to duplicate the field name.

**Comment by alirezanet** [comment]
Hi @asafblum, 
Have you tried the [CustomOperator](https://alirezanet.github.io/Gridify/guide/filtering.html#custom-operators) feature? maybe you can add it the way suits your project?
or could you provide an example (some test code to demonstrate the use case)? It helps for adding new features if currently is not possible.

**Reply by alirezanet** [reply]
I've added an example to a similar question here: https://github.com/alirezanet/Gridify/discussions/133

**Comment by alirezanet** [comment]
Hi @asafblum,
If you're still interested in this feature I should note with some enhancement to the CustomOperators it is now possible in `v2.13.1`,

More info: #135
https://alirezanet.github.io/Gridify/guide/filtering.html#custom-operators
